dev-util/ebuildtester/metadata.xml

```diff
@@ -4,6 +4,14 @@
 	<maintainer type="person">
 		<email>nicolasbock@gentoo.org</email>
 	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>davidroman96@gmail.com</email>
+		<name>David Roman</name>
```

Comment:
I would also suggest doing this in a third commit, not the "add" one. Among other things, this makes it easier to trivially revert, or to cherry-pick.

Reply:
done :)